.github/workflows/test.yml

```diff
@@ -0,0 +1,27 @@
+name: Terraform Test
+
+on:
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    name: Integration test
+    strategy:
+      fail-fast: false
+      max-parallel: 1
```

Comment:
since these are integration tests, I wanted to avoid hammering cloud.gov with a bunch of parallel runs. We may want to increase this a bit though, especially as the database and redis modules come under test, since they take a while to create